NeoML/include/NeoML/Dnn/Layers/GELULayer.h

```diff
@@ -20,25 +20,54 @@ limitations under the License.
 
 namespace NeoML {
 
-// Activation layer with formual x * Ф(x),
-// where Ф(x) - cumulative distribution function for the normal distribution N(0, 1)
-// This layer uses next approximation: x * sigmoid(1.702 * x)
+// Activation layer with formula x * Ф(x),
+// where Ф(x) - cumulative distribution function of the standard normal distribution N(0, 1)
 class NEOML_API CGELULayer : public CBaseLayer {
 	NEOML_DNN_LAYER( CGELULayer )
 public:
 	explicit CGELULayer( IMathEngine& mathEngine );
 
 	void Serialize( CArchive& archive ) override;
 
+	// CDF can be calculated using the error function (slow) or using an approximation. The precise method is used by default.
+	enum TCalculationMode {
+		// x * 0.5( 1 + erf( x / sqrt(2) ) )
+		CM_Precise,
+		// x * sigmoid(1.702x)
+		CM_FastApproximate
```

Comment:
I recommend mention the exact method in the `enum` e.g. `SigmoidApproximate` (in case of future changes, because `tanh` approximation also exists)

Reply:
I prepared `SlowApproximate` for that case. But `SigmoidApproximate`\ `TanhApproximate `seems to be better...

Reply:
TODO: comments and docs